pype/tools/settings/settings/gui_schemas/projects_schema/schema_main.json

```diff
@@ -2,73 +2,71 @@
     "key": "project",
     "type": "dict",
     "children": [
+    {
```

Comment:
Please do not mess up with indentantion it helps to see what is child of what...

Reply:
That's @mkolar formatting, I am guessing it comes from some plugin, we should decide what kind of formatting should be used or we will be moving it there and back again.